codesign/lib/src/google_cloud_storage.dart

```diff
@@ -9,67 +9,41 @@ import 'package:process/process.dart';
 
 import './utils.dart';
 
-/// Utility function class to handle upload/download of files from/to google cloud.
+/// A service to interact with google cloud storage through gsutil.
 class GoogleCloudStorage {
-  GoogleCloudStorage();
-
-  String gsCloudBaseUrl = r'gs://flutter_infra_release';
-
-  /// Wrapper function to upload code signed flutter engine artifact to google cloud bucket.
+  GoogleCloudStorage({
+    required this.processManager,
+    required this.rootDirectory,
+    required this.commitHash,
+  });
+
+  ProcessManager processManager;
```

Comment:
These fields should be final. It looks like the analyzer failed to catch this. Can you try rebasing your branch with main?

https://dart-lang.github.io/linter/lints/prefer_final_fields.html

Reply:
Sure, rebased branch with main, also added finals keywords.